virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-hot-plugging-virtual-disks"]
+= Hot-plugging virtual disks
+include::modules/virt-document-attributes.adoc[]
+:context: virt-hot-plugging-virtual-disks
+
+:FeatureName: Hot-plugging virtual disks
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+toc::[]
+
+If you _hot-plug_ a virtual disk, you can attach or detach the virtual disk to or from a virtual machine instance while a virtual machine is running.
+
+When you _hot-unplug_ a virtual disk, you can no longer  attach or detach the virtual disk to or from a virtual machine instance while a virtual machine is running.
+
+Only hot-plugged disks can be attached or detached while a virtual machine is running, as opposed to virtual disks that are specified in a virtual machine definition, which cannot be attached or detached before you stop the virtual machine or virtual machine instance.
```

Comment:
So disks specified in the VM definition are by definition attached to the VM, you just cannot detach them with a hot-unplug.

Reply:
After I rewrote the text in the previous comment, I decided that this paragraph ("only hot-plugged....") was just introducing confusion. If you don't think so, let me know. Trying to keep it as simple as possible....